### PR TITLE
623 Re-arrange account and profile settings, 622 Add contactEmail, contactPhone to currentUser queries

### PR DESCRIPTION
**Pull request by lorenjohnson**

### CODE REVIEW

File: src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss [hunk 1]
```diff
@@ -95,7 +95,8 @@
 
 label {
   color: $color-regent-70;
-  margin-left: 10px;
+  // LEJ: Commenting-out as has global impact
```
Comment (tibetsprague) [inline src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss:98]: why not just change 'label' to a classname?

File: src/util/navigation.js [hunk 2]
```diff
@@ -180,6 +184,15 @@ export function newMessageUrl () {
   return `${messagesUrl()}/new`
 }
 
+export function messageThreadUrl (person) {
+  // TODO: messageThreadId doesn't seem to be currently ever coming-in from the backend
```
Comment (tibetsprague) [inline src/util/navigation.js:188]: what does this do on the front-end? file a ticket?

File: src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss (hunk 1 above)
Comment (lorenjohnson) [reply src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss:98]: Yes, certainly. That does require making updating to this feature area and doing a more comprehensive review of the impact of removing the global style throughout, but yes I can and will do that.

File: src/util/navigation.js (hunk 2 above)
Comment (lorenjohnson) [reply src/util/navigation.js:188]: https://github.com/Hylozoic/hylo-evo/issues/655

File: src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss (hunk 1 above)
Comment (tibetsprague) [reply src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss:98]: hmm yeah, fair enough, would rather get this out right now

**Review by tibetsprague — APPROVED** [review]

File: src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss (hunk 1 above)
Comment (lorenjohnson) [reply src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss:98]: NOTE: If this is going out as is, previous to the profile design update (in progress), Skill & Interests won't currently be appearing anywhere on the Profile.

I'm ok with that, the design update won't be far behind, but worth pointing-out.

---
... and yeah, :) basically that was the logic I was following as well. I did make better a few things in the course of this branch as it was but needed to stop there.

**Comment by lorenjohnson** [comment]
@tibetsprague with your permission I'd like to add skill tags under bio as in this screenshot to this branch to make it more releasable as-is. This is only a few lines diff and a snapshot update on the PR:

Screen Shot 2020-10-09 at 8 54 32 AM

**Comment by tibetsprague** [comment]
Love it! Go go go.

**Comment by lorenjohnson** [comment]
> Love it! Go go go.

Ok it's in there now. Here is the remaining diff / final commit if you want to give it a quick look. Otherwise, now I think this would be a happy little on-the-way update. https://github.com/Hylozoic/hylo-evo/pull/654/commits/160dcd289dd54c7aac8ebca4e36e1afa94e688dc

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**